docs/components/base/headings.tsx

```diff
@@ -12,9 +12,9 @@ const attributes = {
 
 const STYLES_H1 = css`
   ${h1}
-  margin-top: 0.5rem;
-  margin-bottom: 1rem;
-  padding-bottom: 0.5rem;
+  margin-top: 0;
```

Comment:
```suggestion
  margin-top: ${spacing[2]}px;
```
Adding more room above the title adds white space, which draws a little more attention to the h1.

Here are some screenshots of the changes I am making to this file below. What do you think?

<img width="1486" alt="Screen Shot 2022-08-26 at 1 11 02 AM" src="https://user-images.githubusercontent.com/6455018/186826870-5ed706aa-15b2-441c-8916-72e16da2f26b.png">
<img width="1496" alt="Screen Shot 2022-08-26 at 1 10 54 AM" src="https://user-images.githubusercontent.com/6455018/186826883-339ae721-0378-40aa-8470-59a4c0f1d98e.png">